exercises/etl/etl_test.rb

```diff
@@ -32,27 +41,67 @@ def test_more_keys
     assert_equal expected, ETL.transform(old)
   end
 
-  def test_full_dataset
+  def test_multiple_scores_with_differing_numbers_of_letters
     skip
     old = {
-      1 => %w(A E I O U L N R S T),
-      2 => %w(D G),
-      3 => %w(B C M P),
-      4 => %w(F H V W Y),
-      5 => %w(K),
-      8 => %w(J X),
-      10 => %w(Q Z)
```

Comment:
> Personally I think it's best if all tests just use one syntax or another

Yes! Agreed :+1:

>  I can definitely switch everything to the %w() syntax though if that's the consensus.

Personally, I think the longer examples are a bit easier to read when using `%w()`, but I'm not sure if everyone else will agree with that. This is likely a question of preference, so a weigh in from @Insti and @kotp would be great. I don't think its much of a big deal though 😄